PhD Thesis/Thesis.tex

```diff
@@ -15,8 +15,7 @@
 %	PACKAGES AND OTHER DOCUMENT CONFIGURATIONS (do not modify)
```

Comment:
We are modifying this, so the `(do not modify)` doesn't make much sense. Maybe it should say (in a new line),
> Please make all modifications in the corresponding chapter files and include them here.

also: the comment block above was clearly written for the temporary thesis and should be modified to explain that the file can be switched between temporary and final versions.

Reply:
I've commented elsewhere, but it would be nice to bring in the stuff from `my definitions.tex` here and we could have two sections, one `do not modify` for the packages necessary for the template to run and one `add your packages`.